### DISCUSSION TITLE
使用longtable跨页表格如何改变字号大小？

### DISCUSSION THREAD

**Discussion by geniusxxx** [tuna/thuthesis#967] [Q&A]
按这种方式无法更改跨页表格字号大小，请问是否还有其他方法？
`{
\tiny
\begin{longtable}
...
\end{longtable}
}`

**Comment by zepinglee** [comment]
表格的字号由 `\thu@table@font` 控制。这部分在研究生的《指南》中有要求，所以不建议进行肉眼可见级别的修改。

https://github.com/tuna/thuthesis/blob/60661f7d140c192418cc9151b54a6c59a99fae23/thuthesis.dtx#L3745-L3749

https://github.com/tuna/thuthesis/blob/60661f7d140c192418cc9151b54a6c59a99fae23/thuthesis.dtx#L3708-L3730
